packages/structures/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+	"$schema": "https://json.schemastore.org/tsconfig.json",
+	"extends": "../../tsconfig.json",
+	"include": ["src/**/*.ts", "src/**/*.js", "src/**/*.cjs", "src/**/*.mjs", "bin"],
+	"exclude": ["node_modules"],
+	"compilerOptions": {
+		"experimentalDecorators": false
```

Comment:
Why is this here? Surely it's not `true` by default?

Reply:
No, but it is true in our base tsconfig.json which this extends.